netcontrol/nft.py

```diff
@@ -33,5 +34,59 @@ def _execute_nft_cmd(self, cmd: str) -> dict:
             raise NftablesException(rc, error)
         return json.loads(output)["nftables"]
 
+    def _add_vpn(self, mark: int, vpn: str) -> None:
+        """Adds a VPN to the nftables rule
+
+        Args:
+            mark (int): mark value
+            vpn (str): VPN routing table name
+        """
+        try:
+            self._execute_nft_cmd(f"add rule insalan netcontrol-forward fwmark {mark} lookup {vpn}")
```

Comment:
iprules aren't managed byt `nft` but by `ip rule` (cf [`iprules.sh`](https://github.com/InsaLan/scripts-reseau/blob/main/ip_rules_mini.sh))

Reply:
ohh okay, i clearly misunderstood the assignment :joy: